### ISSUE TITLE
humanize_error function returns type error

### ISSUE THREAD

**Issue by LeiLu199** [alecthomas/voluptuous#269]
```python
import voluptuous
import voluptuous.humanize
import voluptuous.error

allowed_value = [1, 2, 3]



base = voluptuous.Schema({
    voluptuous.Required('age'): int,
    voluptuous.Required('children'): voluptuous.All(voluptuous.ExactSequence([
        {voluptuous.Required('first'): [voluptuous.Any(*allowed_value)]},
        {voluptuous.Required('second'): [voluptuous.Any(*allowed_value)]},
        ]),
        voluptuous.Length(min = 2, max=2))
        })



test = {'age': 40, 'children': [{'first': 12, 'second': 8}]}

def main():
    try:
        base(test)
    except voluptuous.error.Error as e:
        return voluptuous.humanize.humanize_error(test, e)

main()


will  returning traceback 
  File "./Desktop/voluptuous_test.py", line 29, in main
    return voluptuous.humanize.humanize_error(test, e)
  File "/usr/local/lib/python2.7/site-packages/voluptuous/humanize.py", line 26, in humanize_error
    for sub_error in validation_error.errors
  File "/usr/local/lib/python2.7/site-packages/voluptuous/humanize.py", line 26, in <genexpr>
    for sub_error in validation_error.errors
  File "/usr/local/lib/python2.7/site-packages/voluptuous/humanize.py", line 29, in humanize_error
    offending_item_summary = repr(_nested_getitem(data, validation_error.path))
  File "/usr/local/lib/python2.7/site-packages/voluptuous/humanize.py", line 11, in _nested_getitem
    data = data[item_index]
TypeError: list indices must be integers, not str
```

**Comment by alecthomas** [comment]
This was fixed in a30f47fbe74b572d81040c231c792dcba2802611. What version are you using?

**Comment by LeiLu199** [comment]
Hi Alec,
I am using version 0.9.3. Is this commit included in this version?

Best,
Lei

**Comment by spacegaier** [comment]
Fixed as of version 0.10.2 => closing.
